### DISCUSSION TITLE
可悲…

### DISCUSSION THREAD

**Discussion by wedxz** [WeNeedHome/SummaryOfLoanSuspension#873] [General]
首先我不是烂尾楼的业主。我只是觉得这些人太悲惨了，国家竟然允许这样的事情发生.
今天我在群里讨论这个事情，一个烂尾楼的业主居然跟我说，烂不烂尾不重要，说为什么要把这些东西发到GitHub，说为什么要让别的国家人知道。
真是惊到我了，好一个做韭菜的修养。真的觉的他们的遭遇都是他们应得的，不怪镰刀太锋利。

**Comment by guoyongchang** [comment]
我个人觉得这事，大概率得让外国媒体下场（已经有很多人在微博上求美国大使馆关注改事件了）在国际上让领导们面上无光，领导们可能才会考虑下场解决这些问题，或产生问题的人。

**Reply by wedxz** [reply]
> 我个人觉得这事，大概率得让外国媒体下场（已经有很多人在微博上求美国大使馆关注改事件了）在国际上让领导们面上无光，领导们可能才会考虑下场解决这些问题，或产生问题的人。

已经下场了
